addons/docs/src/blocks/mdx.tsx

```diff
@@ -111,7 +111,7 @@ export const AnchorMdx: FC<AnchorMdxProps> = (props) => {
   }
 
   // External URL dont need any modification.
-  return <A {...props} />;
+  return <A target="_blank" {...props} />;
```

Comment:
There are two, separate changes in this PR.

The first, on line 94, updates the "is it an internal link?" heuristic in a helpful manner.

But this change coerces all external links to open in a new window, which we would consider a breaking change. If an author wants the link to open externally they can either continue adding `target="_blank"` to their links or, using the same override mechanism that would enable them to opt out of this change, they can opt _in_ instead.

In other words, please revert this part of your PR, and the first part is great.

Reply:
> But this change coerces all external links to open in a new window, which we would consider a breaking change.

A counterpoint to this is that because of the previous heuristic, the only way a link would be considered external is if it had `target="_blank"`, which is why I thought adding this would keep the behavior as similar as possible. In my mind, this isn't a breaking change since before this there was no way to have an external link that _didn't_ open on a new window.

However, I do think it's an opinionated change, so I totally understand if you prefer to have it opt-in rather than opt-out. I've reverted it already! 😄

Reply:
That's a good point. You're absolutely right that it's an "opinionated change", not a "breaking" one. Thanks for understanding the intent, even when I didn't communicate clearly!